### ISSUE TITLE
EncNet for Cifar-10 Classification

### ISSUE THREAD

**Issue by lmz123321** [zhanghang1989/PyTorch-Encoding#359]
Thx for your great work!
I am currently trying to use EncNet(context encoding layer based ResNet-50) on Cifar-10.
However, I notive your docs on 'recognition' is all about ResNest, Where can I find code for EncNet recognition?
Thank you @zhanghang1989

**Comment by zhanghang1989** [comment]
Hi, thanks for interest in this work. Please refer to the Torch implementation for experiments on CIFAR https://github.com/zhanghang1989/Torch-Encoding-Layer

**Comment by lmz123321** [comment]
Thank you for reply. During waiting, I tried to implement EncNet for CIFAR-10 based on EncNet for segmentation.

I only change the 1*1 conv before Encoding(), from (inchannels, inchannels) to (inchannles, inchannels // 4), as you stated in your paper.

I didn't make any change on 'apply a dropout like regulation on s_{k}',  Does the trick matter a lot ?

**Comment by zhanghang1989** [comment]
You may use `EncodingDrop` layer at https://github.com/zhanghang1989/PyTorch-Encoding/blob/master/encoding/nn/encoding.py#L115

It will apply the dropout like regularization automatically. It improves the results for longer training.

**Comment by lmz123321** [comment]
I see! Thank you again. Problem solved
